static/shared/js/emoji.js

```diff
@@ -1,25 +1,32 @@
-const emoji_codes = require("../generated/emoji/emoji_codes.json");
-
-const util = require("./util");
+import emoji_codes from "../../generated/emoji/emoji_codes.json";
```

Comment:
We can’t import from outside `shared` in `shared`. I’ve just added an ESLint rule to catch this.

Reply:
Ok, I now pass in emoji_codes from ui_init, as we discussed on chat.